aws/src/main/python/cloudwatchAllInstances/lambda_function.py

```diff
@@ -103,39 +105,39 @@ def get_running_instance_for(region):
         instance_list = []
         for reservation in reservations:
             instances = safe_get(reservation, 'Instances')
-            instance_list += [convert_to_ec2_instance_from(instance, region) for instance in instances]
+            instance_list += [convert_to_ec2_instance_from(instance) for instance in instances]
         return instance_list
     else:
         logger.info('0 running instances found in ' + region)
         return []
 
-def convert_to_ec2_instance_from(response_instance, region):
+def convert_to_ec2_instance_from(response_instance):
     logger.info('Convert to EC2Instance from ' + str(response_instance))
     email = None
     tags = response_instance.get('Tags')
-    if tags is not None:
+    if tags != None:
         for tag in tags:
             if tag['Key'] == 'GitUserEmail':
                 email = tag['Value']
     instance_id = safe_get(response_instance, 'InstanceId')
-    #Email not found in tags, so now check instance in directly started instances
+    #Email not found in tags, so now check for directly started instances
     username=None
     if email is None:
-        cloudtrail = boto3.client('cloudtrail', region_name=region)
```

Comment:
@rajnikantsh This was a change from a copy in the server. I think we still need to do this from a region perspective, otherwise it uses what's in the environment - afaik. But this is from your change, so I wanted to check

Reply:
yes @JustinPihony we need it region specific